### PR TITLE
Add container minutes 2023-06-26

### PR DESCRIPTION
**Pull request by jschoone**

### CODE REVIEW

File: container/20230626.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- @bitkeks
+- @wieneo
+- @aniruddha
+
+## Community call and minutes transfer
+  - @jschoone
+
+
+## Welcome
+- Aniruddha (Syself), ClusterAPI maintainer
```
Comment (batistein) [inline container/20230626.md:28]: Nitish (Syself), expert in writing operators and cloud-native software

Comment (batistein) [inline container/20230626.md:28]: Aniruddha (Syself), Cloud Engineer and Cluster API maintainer

File: container/20230626.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- @joshmue
+- @chess-knight
+- @jschoone
+- @saschascherrer
+- @matofeder
+- @janiskemper
+- @bitkeks
+- @wieneo
+- @aniruddha
+
```
Comment (batistein) [inline container/20230626.md:22]: - `@nitisht`

**Review by batistein — CHANGES_REQUESTED** [review]

**Review by batistein — APPROVED** [review]

File: container/20230626.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- Cluster Stack Example Docker [issues/#348](https://github.com/SovereignCloudStack/issues/issues/348)
+- Document how to implement Cluster Stacks [issues/349](https://github.com/SovereignCloudStack/issues/issues/349)
+- Provide example cluster stacks [issues/#322](https://github.com/SovereignCloudStack/issues/issues/322)
+- How to contribute?
+- Evaluating KubeVirt
+
+##### Lightning Talk about ClusterStacks next week (2023-07-03)
+
+### Lightning talk Cilium
```
Comment (guettli) [inline container/20230626.md:64]: @jschoone Lightning Talk about ClusterStacks next week (2023-07-03) ? Something is wrong here, or do you want me to give this talk a second time?

Comment (jschoone) [reply container/20230626.md:64]: No, thats the talk about the ClusterStacks from Janis afaik

**Pull request merged**